src/modules/imselector/imselector.h

```diff
@@ -25,7 +28,26 @@ FCITX_CONFIGURATION(
         "TriggerKeyLocal",
         _("Trigger Key for only current input context"),
         {},
-        KeyListConstrain()};);
+        KeyListConstrain()};
+    KeyListOptionWithToolTip switchKey{
+        this,
+        "SwitchKey",
+        _("Hotkey for switching to the Nth(0-based) input method"),
+        {},
+        KeyListConstrain(),
+        {},
+        ToolTipAnnotation(
+            _("Press the '+' button to bind a hotkey for next input method"))};
```

Comment:
You may explain the n-th key in this option will switch to the n-th input method here.

Reply:
Done.